src/server/ua_subscription.h

```diff
@@ -57,7 +59,7 @@ typedef struct UA_Notification {
 
     /* See the monitoredItemType of the MonitoredItem */
     union {
-        UA_Event event;
+        UA_EventNotification *event;
```

Comment:
Does this have to be a pointer?
Or can we directly add the EventNotification here?